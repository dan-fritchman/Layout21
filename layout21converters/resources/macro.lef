VERSION 5.4 ; 
UNITS
    DATABASE MICRONS 2000 ;
END UNITS
MACRO macro_name
    CLASS BLOCK ;
    SIZE 999.9 BY 111.1 ;
    SYMMETRY X Y R90 ;
    PIN pin_name
        DIRECTION INPUT ;
        PORT
            LAYER layer_name ;
                RECT  88.4 0.0 88.78 1.06 ;
        END
    END pin_name
END macro_name
END LIBRARY