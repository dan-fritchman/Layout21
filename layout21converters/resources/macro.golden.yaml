---
macros:
  - name: macro_name
    pins:
      - name: pin_name
        ports:
          - layers:
              - layer_name: layer_name
                geometries:
                  - Shape:
                      Rect:
                        - x: "88.4"
                          y: "0.0"
                        - x: "88.78"
                          y: "1.06"
        direction: Input
    class:
      Block:
        tp: ~
    size:
      - "999.9"
      - "111.1"
    symmetry:
      - X
      - Y
      - R90
version: "5.4"
units:
  database_microns: 2000
